ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java

```diff
@@ -55,7 +62,8 @@ public void addUserTest() throws Exception {
             assertTrue(manageUserPage.getUserSavedNotification().contains("User Saved"));
 
             homePage = new HomePage(goToLoginPage().login("super_nurse", "Nurse123"));
-            homePage.getLoggedUsername().contains("super_nurse");
+            homePage.waitForPage();
+            assertTrue(homePage.containsText("super_nurse"));
```

Comment:
After trying different element attributes for getting the `loggedUsername`, all the element property values (`css, id, linkText, xpath` )  break only with chrome `headless=true`. Its from this reason that we rather assert the presence of the text `super_nurse` on the page since its equivalent to  `homePage.getLoggedUsername().contains("super_nurse")`